app/forms/fields.py

```diff
@@ -142,6 +142,7 @@ class CustomIntegerField(IntegerField):
     """
     def __init__(self, **kwargs):
         super(CustomIntegerField, self).__init__(**kwargs)
+        self.data = None
```

Comment:
Pylint complained that self.data is used later in this class but not declared in `__init__`